go.mod

```
@@ -1,8 +1,8 @@
 module github.com/cli/cli/v2
 
-go 1.22.0
+go 1.22.5
 
-toolchain go1.22.5
+toolchain go1.22.6
```

Since this PR was created, [`1.22.7` was released](https://go.dev/doc/devel/release#go1.22.minor):

> go1.22.7 (released 2024-09-05) includes security fixes to the encoding/gob, go/build/constraint, and go/parser packages, as well as bug fixes to the fix command and the runtime. See the [Go 1.22.7 milestone](https://github.com/golang/go/issues?q=milestone%3AGo1.22.7+label%3ACherryPickApproved) on our issue tracker for details.

Is this worth bumping to?

```suggestion
toolchain go1.22.7
```